index.js

```diff
@@ -0,0 +1,57 @@
+function checkSalaryTotal(lineup) {
+  return lineup.reduce((salary, player) => {
+    return salary + player.salary
+  }, 0)
+}
+
+function checkTeamId(lineup) {
+  return lineup.reduce((teams, player) => {
+    teams[player.teamId] = teams[player.teamId] === undefined ? 1 : teams[player.teamId] + 1
+
+    return teams
+  }, {})
+}
```

Comment:
This function is reformatting, not checking. `getCountsByTeamId` or something would be a better name